src/libraries/System.Security.Cryptography.Algorithms/src/Internal/Cryptography/Pbkdf2Implementation.Windows.cs

```diff
@@ -18,7 +18,7 @@ internal static partial class Pbkdf2Implementation
     {
         // For Windows 7 we will use BCryptDeriveKeyPBKDF2. For Windows 8+ we will use BCryptKeyDerivation
         // since it has better performance.
```

Comment:
Since we got this wrong, I recommend we augment the comment.

```suggestion
        // For Windows 7 we will use BCryptDeriveKeyPBKDF2. For Windows 8+ (seen as version 6.2.0) we will
        // use BCryptKeyDerivation since it has better performance.
```